### PR TITLE
Some improvements on templates

### PR DESCRIPTION
**Pull request by Keats**

### CODE REVIEW

File: content/about/index.md [hunk 1]
```diff
@@ -1,6 +1,5 @@
 +++
 title = "About"
-template = "page.html"
```
Comment (Keats) [inline content/about/index.md:3]: page.html is the default template, no need to specify it.

For about and subscribe, if you don't want to put content next to it, there is no need to put them in a directory. This file is equivalent to `content/about.md` for example.

File: templates/index.html [hunk 2]
```diff
@@ -1,61 +1,60 @@
...
-  	<meta property="og:description" content="{{ config.description }}">
-  	<meta property="og:title" content="{{ config.title }}">
-  	<meta property="og:image" content="{{ get_url(path='images/top-banner.png') }}">
-  	<!-- /Facebook -->
+		{% set social_title = config.title %}
+		{% if page %}
+			{% set social_title =  page.title ~ " | " ~ config.title %}
+		{% elif section and section.title %}
+			{% set social_title =  section.title ~ " | " ~ config.title %}
+		{% endif %}
```
Comment (Keats) [inline templates/index.html:23]: This way we don't have to repeat the meta on pages and section, it's only defined in the index.

File: templates/index.html [hunk 3]
```diff
@@ -1,61 +1,60 @@
...
 						<li><a href="{{config.base_url}}">Home</a></li>
 						<span> | </span>
-						<li><a href="{{config.base_url}}/episodes">Episodes</a></li>
+						<li><a href="{{ get_url(path="@/episodes/_index.md") }}">Episodes</a></li>
 						<span> | </span>
-						<li><a href="{{config.base_url}}/subscribe">Subscribe</a></li>
+						<li><a href="{{ get_url(path="@/subscribe/index.md") }}">Subscribe</a></li>
 						<span> | </span>
-						<li><a href="{{config.base_url}}/about">About</a></li>
+						<li><a href="{{ get_url(path="@/about/index.md") }}">About</a></li>
```
Comment (Keats) [inline templates/index.html:57]: it isn't super necessary to do links like that in templates but I find it helpful when some pages define their own URL path that are different from the file paths and I want to know where this content is

**Review by Keats — COMMENTED** [review]
I've added some `| safe` as well to avoid some character escaping, maybe that will help with the Twitter image?

For the transcript, I'm not sure what's the best way to handle it. You could make some fancy styling if you have it in a file and load it with https://www.getzola.org/documentation/templates/overview/#load-data.
You could make a shortcode like `{{ transcript(path=.., person1="Jeremy", person2="Vincent")}}` and if the data is in an iteratable format have the name added automatically with some styling.

File: content/about/index.md (hunk 1 above)
Comment (jeremyjung) [reply content/about/index.md:3]: Got it.  Since I don't have any content so far I'll rename the md files.

File: templates/index.html (hunk 3 above)
Comment (jeremyjung) [reply templates/index.html:57]: Yeah, this does look better. So you would only use `config.base_url` for linking to the root path?

File: templates/index.html (hunk 2 above)
Comment (jeremyjung) [reply templates/index.html:23]: Ah yes, this is much cleaner. Hadn't thought to check if it was on a page or section.

**Comment by jeremyjung** [comment]
> 
> I've added some `| safe` as well to avoid some character escaping, maybe that will help with the Twitter image?
> 

Oddly enough twitter is showing the header images now.  I'm not sure why since nothing changed.  Still, the `| safe` tip is good since it makes the links clickable and easier to read if you're viewing source.

> For the transcript, I'm not sure what's the best way to handle it. You could make some fancy styling if you have it in a file and load it with https://www.getzola.org/documentation/templates/overview/#load-data.
> You could make a shortcode like `{{ transcript(path=.., person1="Jeremy", person2="Vincent")}}` and if the data is in an iteratable format have the name added automatically with some styling.

I will look into load-data and shortcodes.  I think the trouble comes in if I want to add line breaks or make clickable links inside the transcript text. I'm not sure if csv/json/toml would really work for that.

**Pull request merged**